web-api/terraform/api/waf.tf

```diff
@@ -0,0 +1,82 @@
+resource "aws_wafv2_web_acl" "apis" {
+  name  = "apis_${var.environment}_${var.current_color}"
+  scope = "REGIONAL"
+
+  default_action {
+    allow {}
+  }
+
+  rule {
+    name     = "per_ip_request_limit"
+    priority = 1
+
+    action {
+      count {} // change to `block {}` when confident in ruleset
```

Comment:
See https://github.com/ustaxcourt/ef-cms/issues/711 — the plan is to deploy this in count-only mode to ensure we don’t mess this up and block everyone — and once we’re confident that we haven’t blocked normal usage of the system, we’ll switch this & the rule below to `block {}`.